src/components/LogoutButton.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { useContext } from "react";
+import { UserContext } from "../contexts/UserContext";
+
+function LogoutButton() {
+  const { logout } = useContext(UserContext);
```

Comment:
I would say you would only need 1 logout button and it should only show when logged in.